buildozer/targets/android.py

```diff
@@ -99,6 +99,10 @@ def _sdkmanager(self, *args, **kwargs):
         """Call the sdkmanager in our Android SDK with the given arguments."""
         # Use the android-sdk dir as cwd by default
         kwargs['cwd'] = kwargs.get('cwd', self.android_sdk_dir)
+        if "CI" in environ and ">/dev/null" not in args:
+            args = list(args)
+            args.insert(len(args), ' | grep -v "^\\["')
+            args = tuple(args)
         command = self.sdkmanager_path + ' ' + ' '.join(args)
```

Comment:
Instead if this 4 lines of code "adding" the `grep` o the args. I think it would be more simple to go after the `command = self.sdkmanager_path + ' ' + ' '.join(args)` line and to do something like:
```
if 'CI' in environ:
    # hide download lines which look like "[=          ] 10% Downloading platform-tools"
    command += r' | grep -v "^\["'
```
Also note that I've used a raw-string to make the grep expression more readable (by skipping the double escaping of the `\`).
Last I've added a little code comment because it's not super obvious what we are doing